npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -274,7 +287,7 @@ def error_details_from_captures(captures)
             requirement_name:
               captures.fetch("required_dep").sub(/@[^@]+$/, ""),
             requirement_version:
-              captures.fetch("required_dep").split("@").last,
+              captures.fetch("required_dep").split("@").last.gsub('"', ""),
```

Comment:
What's the gsub fixing?

Reply:
ah the requirement is quoted `react@"^16.14.0"`

Reply:
Yeah, this is a bit gnarly but it does the job